spotpass/read-boss-db-3ds.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs-extra');
 const apps = require('./ctr-boss-apps.json');
 
-const NPDL_REGEX = /npdl\.(?:cdn|c\.app)\.nintendowifi\.net\/p01\/nsa\/([A-z0-9]{16})\/(\w*)/g;
+const NPDL_REGEX = /(?:npdl|npfl)\.(?:cdn|c\.app)\.nintendowifi\.net\/p01\/(?:nsa|filelist)\/([A-z0-9]{16})\/(\w*)/g;
```

Comment:
Only one thing. Either this variable needs to be renamed, or the NPFL search needs to be a separate regex. Either way works

Reply:
I just renamed it to `REGEX`. Hope that works.

Reply:
Works for me :+1: